addons/realisticnames/CfgVehicles.hpp

```diff
@@ -363,6 +392,11 @@ class CfgVehicles {
         displayName = CSTRING(Plane_Fighter_03_AA_Name);
     };
 
+    class Plane_Fighter_03_dynamicLoadout_base_F;
+    class I_Plane_Fighter_03_dynamicLoadout_F: Plane_Fighter_03_dynamicLoadout_base_F {
+        displayName = CSTRING(Plane_Fighter_03_Name);
+    };
+
     // uavs
     class UAV_02_base_F;
```

Comment:
Vanilla Config:
`class UAV_02_base_F: UAV {displayName = "MQ-4A Greyhawk";}`
`class B_UAV_02_F: UAV_02_base_F` uses inheritance
`class O_UAV_02_F: UAV_02_base_F {displayName = "K40 Ababil-3";};`
 `class I_UAV_02_F: UAV_02_base_F {displayName = "K40 Ababil-3";}`
So adding it to all is necessary